paddle/fluid/operators/fc_op.cc

```diff
@@ -142,13 +139,20 @@ class FCOpKernel : public framework::OpKernel<T> {
   void Compute(const paddle::framework::ExecutionContext& ctx) const override {
     PADDLE_ENFORCE(platform::is_cpu_place(ctx.GetPlace()),
                    "It must use CPUPlace.");
-    auto input = ctx.Input<Tensor>("Input");
-    auto w = ctx.Input<Tensor>("W");
-    auto bias = ctx.Input<Tensor>("Bias");
-    auto output = ctx.Output<Tensor>("Out");
+    auto input = ctx.Input<framework::LoDTensor>("Input");
+    auto w = ctx.Input<framework::LoDTensor>("W");
+    auto bias = ctx.Input<framework::LoDTensor>("Bias");
+    auto output = ctx.Output<framework::LoDTensor>("Out");
```

Comment:
why change to LodTensor

Reply:
If not change to LodTensor, `output->Resize` and `output->set_lod` could not be used.

Reply:
`w` and `bias` already roll back to use `Tensor`.